### ISSUE TITLE
Login Arguments

### ISSUE THREAD

**Issue by jalbertcory** [rupert/pandora-to-google-music#2]
This may be a version thing with a newer gmusicapi, but if you try to run this with the current version, you get this error.
  File "pandora_to_google_music.py", line 387, in <module>
    main()
  File "pandora_to_google_music.py", line 383, in main
    gmusic_password
  File "pandora_to_google_music.py", line 328, in pandora_to_google_music
    gmusic_client.login(gmusic_email, gmusic_password)
TypeError: login() takes exactly 4 arguments (3 given)

**Comment by rupert** [comment]
In newer versions of `gmusicapi` there's an extra argument, `android_id`, to the the `login` method.

I don't use Pandora anymore (not available in Europe) but I'll see if I can fix it.

**Comment by Talyrius** [comment]
I changed line 328 to:
```python
    gmusic_client.login("en_US", gmusic_email, gmusic_password)
```

However, I now receive a login exception for Pandora:
> Traceback (most recent call last):
> &nbsp;&nbsp;File "pandora_to_google_music.py", line 387, in <module>
> &nbsp;&nbsp;&nbsp;&nbsp;main()
> &nbsp;&nbsp;File "pandora_to_google_music.py", line 383, in main
> &nbsp;&nbsp;&nbsp;&nbsp;gmusic_password
> &nbsp;&nbsp;File "pandora_to_google_music.py", line 330, in pandora_to_google_music
> &nbsp;&nbsp;&nbsp;&nbsp;pandora_client = PandoraClient(pandora_email, pandora_password)
> &nbsp;&nbsp;File "pandora_to_google_music.py", line 31, in \_\_init__
> &nbsp;&nbsp;&nbsp;&nbsp;raise LoginException("Pandora login failed, check email and password")
> \_\_main__.LoginException: Pandora login failed, check email and password

I've verified that the login details are correct. I tried changing the password and I even eliminated all special characters while I was at it.

**Comment by Talyrius** [comment]
> I don't use Pandora anymore (not available in Europe) but I'll see if I can fix it.

There are free VPNs available that impose speed and/or data limitations that should suffice for testing purposes.

I'm unfamiliar with Python and the API being used in this project. However, if there's anything I can do to assist, please let me know.

**Comment by rupert** [comment]
Fixed by #3, try the latest version.
